packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -366,3 +366,17 @@ body.theme-conversionxl-better {
     font-size: var(--lumo-font-size-l);
   }
 }
+
+/**
+ * Add spacing and typography styles to details and summary elements
+ * @see https://cxlworld.slack.com/archives/C01HXU5GGJG/p1724673491752389
+ * @since 2024.08.26
+ */
+details {
```

Comment:
Hmmm, are we not able to build a quick block editor `details` -> `vaadin-details` element converter instead, in spirit of https://github.com/conversionxl/cxl-section-block

Reply:
Today we'll use ITMW approach but created a task for the future: https://app.clickup.com/t/86b1wx7r9